docker/build-debs.sh

```diff
@@ -78,7 +87,12 @@ fi
 
 # Install build dependencies and build package
 echo "y" | sudo mk-build-deps --install
+
```

Comment:
This had to be done otherwise following `dpkg-buildpackage` complained that JAVA_HOME can not be set like it was set at the beginning of this file.